include/nbl/builtin/hlsl/subgroup/fft.hlsl

```diff
@@ -32,6 +32,7 @@ template<typename Scalar, uint32_t SubgroupSize, bool inverse>
 void FFT(NBL_REF_ARG(complex_t<Scalar>) lo, NBL_REF_ARG(complex_t<Scalar>) hi) 
 {
     const uint32_t doubleSubgroupSize = SubgroupSize << 1;
+    
     // special first iteration
     if (! inverse)
         fft::DIF<Scalar>::radix2(fft::twiddle<Scalar, false>(glsl::gl_SubgroupID(), doubleSubgroupSize), lo, hi);                                                                                   
```

Comment:
gl_SiubgroupID is not the ordinal of the invocation in the subgroup, but the index of the whole subgroup, did you mean to do that?

Reply:
P.S. one more thing about the code later on, you should divide `hi` by scalar instead of dividing by complex, it will be faster (just how you divide `lo`)

Reply:
whoops, I thought I had changed that back, I was testing that the operator overloading was working properly there.